src/zope/app/onlinehelp/onlinehelptopic.py

```diff
@@ -153,20 +158,24 @@ def __init__(self, id, title, path, parentPath, interface=None, view=None):
 
         super(BaseOnlineHelpTopic, self).__init__()
 
+    def _newContainerData(self):
+        # Ensure consistent iteration order for tests.
+        from collections import OrderedDict
+        return OrderedDict()
```

Comment:
Hmm, this can apparently cause issues with security checkers in Python 3:
```
  File "//lib/python3.5/site-packages/zope/app/tree/node.py", line 95, in getChildObjects
    children = self._get_child_objects_adapter().getChildObjects()
  File "/lib/python3.5/site-packages/zope/app/tree/adapters.py", line 94, in getChildObjects
    return list(self.context.values()) if self.hasChildren() else []
zope.security.interfaces.ForbiddenAttribute: ('__iter__', odict_values([<zope.app.onlinehelp.onlinehelptopic.RESTOnlineHelpTopic object at 0x10f87aac8>, <zope.app.onlinehelp.onlinehelptopic.RESTOnlineHelpTopic object at 0x10f87a128>]))
```